libs/eo/faq/shell/src/lib/eo-faq-shell.component.ts

```diff
@@ -24,43 +24,48 @@ import { eoFaqRoutePath } from '@energinet-datahub/eo/faq/routing';
     `
       :host {
         display: block;
-        width: 800px; // Magic number by designer
+        width: 1040px; // Magic number by designer
       }
 
-      h3 {
+      .faq-link a {
+        color: var(--watt-color-primary);
+        text-decoration: none;
+      }
+
+      h2 {
         margin-top: var(--watt-space-xl);
         margin-bottom: var(--watt-space-s);
       }
     `,
   ],
   template: `
```

Comment:
Remember that you will be formatting **a** tags from now and until the end of days, if you do not handle this in the build steps :-)

Reply:
What do you mean? 😄

Reply:
Linting is splitting the tags up during build - This is automated. So manual updates, will be reverted on the next build where linting will kick in again.

So, you should update the linting rules in the build pipeline, and not the tags in the templates :-)

Reply:
I will find a linter rule in the future to avoid having this as a topic :P